src/schema/event/resolvers.ts

```diff
@@ -1,7 +1,7 @@
-import Promise from 'promise';
-
 import composeQuery from '../../utils/composeQuery';
 import normalizeKeys from '../../utils/normalizeKeys';
+import { EventParams } from './types';
```

Comment:
You can also import this from generated types and get rid of extra EventParams type